Web/Scotty/Internal/Types.hs

```diff
@@ -51,6 +52,16 @@ data Options = Options { verbose :: Int -- ^ 0 = silent, 1(def) = startup banner
 instance Default Options where
     def = Options 1 defaultSettings
 
+newtype RouteOptions = RouteOptions { maxRequestSize :: Maybe Kilobytes -- max allowed request size in KB
```

Comment:
Can we make this field name consistent with the other changes, i.e. `maxRequestBodySize`.

Reply:
> Can we make this field name consistent with the other changes, i.e. `maxRequestBodySize`.

done